{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 8 Advanced Virtualization",
      "fix_state": "Affected",
      "package_name": "qemu-kvm",
      "cpe": "cpe:/a:redhat:advanced_virtualization:8::el8"
    },
    {
      "product_name": "Red Hat OpenStack Platform 10 (Newton)",
      "fix_state": "Affected",
      "package_name": "qemu-kvm-rhev",
      "cpe": "cpe:/a:redhat:openstack:10"
    },
    {
      "product_name": "Red Hat OpenStack Platform 13 (Queens)",
      "fix_state": "Affected",
      "package_name": "qemu-kvm-rhev",
      "cpe": "cpe:/a:redhat:openstack:13"
    },
    {
      "product_name": "Red Hat Enterprise Linux 5",
      "fix_state": "Not affected",
      "package_name": "kvm",
      "cpe": "cpe:/o:redhat:enterprise_linux:5"
    },
    {
      "product_name": "Red Hat Enterprise Linux 5",
      "fix_state": "Not affected",
      "package_name": "xen",
      "cpe": "cpe:/o:redhat:enterprise_linux:5"
    },
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Not affected",
      "package_name": "qemu-kvm",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Not affected",
      "package_name": "qemu-kvm",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Affected",
      "package_name": "qemu-kvm-ma",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Affected",
      "package_name": "qemu-kvm-rhev",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "qemu-kvm",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    }
  ],
  "threat_severity": "Important",
  "public_date": "2020-01-23T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2020-1711 QEMU: block: iscsi: OOB heap access via an unexpected response of iSCSI Server",
    "id": "1794290",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1794290"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "7.7",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:C/C:L/I:L/A:H",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-122",
  "statement": "",
  "acknowledgement": "Red Hat would like to thank Felipe Franciosi (nutanix.com), Raphael Norwitz (nutanix.com), and Peter Turschmid (nutanix.com) for reporting this issue.",
  "name": "CVE-2020-1711",
  "document_distribution": "",
  "details": [
    "An out-of-bounds heap buffer access flaw was found in the way the iSCSI Block driver in QEMU versions 2.12.0 before 4.2.1 handled a response coming from an iSCSI server while checking the status of a Logical Address Block (LBA) in an iscsi_co_block_status() routine. A remote user could use this flaw to crash the QEMU process, resulting in a denial of service or potential execution of arbitrary code with privileges of the QEMU process on the host.",
    "An out-of-bounds heap buffer access flaw was found in the way the iSCSI Block driver in QEMU handled a response coming from an iSCSI server while checking the status of a Logical Address Block (LBA) in an iscsi_co_block_status() routine. A remote user could use this flaw to crash the QEMU process, resulting in a denial of service or potential execution of arbitrary code with privileges of the QEMU process on the host."
  ],
  "references": null
}